notebooks/02_Data_Sync.ipynb

```diff
@@ -53,20 +53,21 @@
     "  inserts, see\n",
```

Comment:
<p>What is meant by team member here? Might want to define client as well.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/spyglass/pull/735/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>